client/src/pages/NewReport/NodesSection/NodesSection.tsx

```diff
@@ -0,0 +1,127 @@
+import './NodesSection.scss';
+import SectionComponent from 'components/SectionComponent/SectionComponent.tsx';
+import Table, { TableColumn } from 'components/Table/Table.tsx';
+import { useState } from 'react';
+import TagButton from 'components/TagButton/TagButton.tsx';
+import { NodeEntry } from 'api/managment-service';
+import SVGIcon from 'components/SVGIcon/SVGIcon.tsx';
+import ActionButton, { ActionButtonColor } from 'components/ActionButton/ActionButton.tsx';
+import OverlayComponent from 'components/OverlayComponent/OverlayComponent.tsx';
+
+const MOCK_APPLICATIONS: NodeEntry[] = [
+    {
+        id: '1',
+        name: 'alerts-api-database',
+        precision: 'high',
+        customPrompt: 'ignore s3 logs...',
+        updated: '07.03.2024 15:32',
+        added: '07.03.2024 15:32',
+    },
+    {
+        id: '2',
+        name: 'alerts-api-backend',
+        precision: 'none',
+        customPrompt: 'ignore s3 logs...',
+        updated: '07.03.2024 15:32',
+        added: '07.03.2024 15:32',
+    },
+];
+
+const NodesSection = () => {
+    const [rows, setRows] = useState<NodeEntry[]>(MOCK_APPLICATIONS);
+    const [showModal, setShowModal] = useState(false);
```

Comment:
Consider adding logic for fetching data from ManagementServiceApi, and mock response there

Reply:
That's for next pr, it would add complexity in this long PR that I wanted to avoid